portal-impl/src/com/liferay/portlet/documentlibrary/service/impl/DLFileEntryLocalServiceImpl.java

```diff
@@ -336,8 +336,15 @@ public DLFileEntry addFileEntry(
 		// Folder
 
 		if (folderId != DLFolderConstants.DEFAULT_PARENT_FOLDER_ID) {
-			_dlFolderLocalService.updateLastPostDate(
-				dlFileEntry.getFolderId(), dlFileEntry.getModifiedDate());
+			DLFolder dlFolder = _dlFolderLocalService.getFolder(
+				dlFileEntry.getFolderId());
+
+			if (!CTCollectionThreadLocal.isProductionMode() &&
+				!dlFolder.isInHiddenFolder()) {
```

Comment:
@brookedalton can you fix this conditional? This wont update the post date in production.